tests/4.5/target_update/test_target_update_if.c

```diff
@@ -71,7 +70,7 @@ for(count=0; count< 4;count++){
   change_flag = init_b();
   #pragma omp target update if (change_flag) to(b[:N]) //update b=4 for all odd iterations
```

Comment:
Is this comment of b=4 accurate?

Reply:
Good catch, I do not believe this is accurate. Will look at this closer and fix.

Reply:
@josemonsalve2 looked into this, b=4 is correct value during "odd" iterations. This is good to go.